src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/AngularDistribution/Tabulated/Builder/src/construct.hpp

```diff
@@ -0,0 +1,18 @@
+Tabulated construct(){
+  equalSize( this->PDF_, "PDF" );
+  equalSize( this->CDF_, "CDF");
```

Comment:
Integrate these in the try ... catch, this way you do not need to catch bad_optional_access in equalSize( ... ).

Reply:
Yeah, but that adds a lot of extra code that I don't want to have to duplicate. That's why I created a function.

Reply:
Refactored because I have the generic `verify::equalSize` function.